### ISSUE TITLE
Default order for a search - decision request to default to last modified

### ISSUE THREAD

**Issue by mparsons-ce** [cassproject/cass-editor#742]
The default order for the search is alphabetic. 
I recommend that the default order by last modified

- The order is minor for an organization with a few frameworks
- If there are many, it makes sense to show the most recently modified ones first
- Similarly, when aligning a framework to assessments, it would be more user friendly to show the one I just updated first

Thoughts?

**Comment by siuc-nate** [comment]
This should also probably be last modified with regard to the CaSS record itself rather than the date in the framework's ceasn:dateModified property.

**Comment by jeannekitchens** [comment]
I think both make sense.  It is fine to default to most recent but there needs to be a filter sort by alpha order as well.

**Comment by Lomilar** [comment]
This isn't a response to the entire issue, however:

Sort options are available in the filter list in the new one and in a drop down in the old one.
![image](https://user-images.githubusercontent.com/337692/100646487-edeae200-32f2-11eb-8a4a-8d65f442a2f6.png)

**Comment by siuc-nate** [comment]
Is there a way to unselect the sort order to go back to alphabetical sorting after clicking one of those (or perhaps alpha sort order should be added to the list of options)?

**Comment by Lomilar** [comment]
Yes, Xing the tag on the left will default back to the original sort order.

**Comment by siuc-nate** [comment]
Not to nitpick, but wouldn't that clear all of the filters? What if the user just wants to change the sort order back?

**Comment by Lomilar** [comment]
Each filter / sort has its own tag. You can remove one at a time.
![image](https://user-images.githubusercontent.com/337692/100653410-36a79880-32fd-11eb-920b-76b41711bd29.png)
